plugin/api.go

```diff
@@ -102,6 +102,11 @@ type API interface {
 	// The status parameter can be: "online", "away", "dnd", or "offline".
 	UpdateUserStatus(userId, status string) (*model.Status, *model.AppError)
 
+	// UpdateUserActive deactivates or reactivates an user.
+	//
+	// Minimum server version: 5.6
```

Comment:
Since 5.6 will ship soon, this PR will go into 5.8. Could you please change this?

Reply:
Updated the version to 5.8